### PR TITLE
add cli example notebooks

### PR DESCRIPTION
**Pull request by kkvtran**
Add CLI notebook examples to the notebooks directory.

### CODE REVIEW

**Review by kaoutar55 — CHANGES_REQUESTED** [review]

**Comment by kaoutar55** [comment]
Hi @kkvtran can you change the directory to - notebooks/Experiment-Runner-examples/

File: notebooks/CLI-examples/cli_cloud_LeNet5_training.ipynb [hunk 1]
```diff
@@ -0,0 +1,1704 @@
...
+   "source": [
+    "DATASET = FashionMNIST\n",
+    "MODEL = create_analog_lenet5_network()\n",
+    "BATCH_SIZE = 8\n",
+    "LOSS_FUNCTION = NLLLoss\n",
+    "EPOCHS = 2\n",
+    "LEARNING_RATE = 0.05\n",
+    "MAX_ELEMENTS = 2000\n",
+    "\n",
+    "my_experiment = BasicTraining(\n",
```
Comment (kaoutar55) [inline notebooks/CLI-examples/cli_cloud_LeNet5_training.ipynb:152]: You call BasicTraining twice in this example. Please remove one instance.

File: notebooks/CLI-examples/cli_cloud_LeNet5_inferencing.ipynb [hunk 2]
```diff
@@ -0,0 +1,1618 @@
...
+    "    inference_repeats = REPEATS,\n",
+    "    inference_time = I_TIMES\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##  Set up required data for the CLI cloud runner"
```
[the remark this answers is not in the document]
Comment (kkvtran) [reply notebooks/CLI-examples/cli_cloud_LeNet5_inferencing.ipynb]: I made the requested changes.

File: notebooks/CLI-examples/cli_cloud_LeNet5_inferencing.ipynb [hunk 3]
```diff
@@ -0,0 +1,1618 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATASET = FashionMNIST\n",
+    "MODEL = create_analog_lenet5_network()\n",
+    "BATCH_SIZE = 8\n",
+    "REPEATS = 2\n",
```
[the remark this answers is not in the document]
Comment (kkvtran) [reply notebooks/CLI-examples/cli_cloud_LeNet5_inferencing.ipynb]: Done.

**Comment by maljoras** [comment]
@kaoutar55, the directory name "Experiment-Runner-Examples" is not in line with the other names also "-" is problematic. What about simply `cli_examples`  or `cli` ?

**Comment by maljoras** [comment]
@kkvtran @kaoutar55 Would be also good to clean up the notebooks directory a bit, like having consistent naming for the files and putting all the "*png" into a separate "data" or so directory, so that it is less cluttered.

**Comment by kkvtran** [comment]
@kaoutar55 @maljoras I made the requested changes. Please review.  Thanks.

**Review by maljoras — APPROVED** [review]
Looks good to me. Thanks @kkvtran

File: notebooks/CLI-examples/cli_cloud_LeNet5_training.ipynb (hunk 1 above)
Comment (kkvtran) [reply notebooks/CLI-examples/cli_cloud_LeNet5_training.ipynb]: Done

**Comment by kkvtran** [comment]
> @kaoutar55, the directory name "Experiment-Runner-Examples" is not in line with the other names. Also, having "-" in a directory name is problematic on linux. What about simply `cli_examples` or `cli` ?

Done.

**Pull request merged**